src/main/java/com/icthh/xm/ms/timeline/config/cassandra/CassandraConfiguration.java

```diff
@@ -117,4 +132,14 @@ public Session session(Cluster cluster) {
         log.debug("Configuring Cassandra session");
         return cluster.connect();
     }
+
+    @Bean
+    public EntityMappingRepository entityMappingRepository(Session session) {
+        return new EntityMappingRepository(session);
+    }
+
+    @Bean
+    public TimelineCassandraRepository timelineRepository(TenantPropertiesService tenantPropertiesService, TenantContextHolder tenantContextHolder, Session session) {
```

Comment:
What is a string length for this line? Please check checkstyle settings and all new code.

Reply:
done